testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
+
+## Thou shalt be consistent within projects
+If a project has a de-facto standard, it should be maintained. The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request.
+
+## Thy tests should be obvious
+If the test behaviour isn't obvious by looking at it, it's no good. If I have to scroll for days to read a test, it's no good.
+
+## Thy tests should be loosely coupled with thy code
```

Comment:
If code is "only the thing being tested" then this is contradictory. IE changes to the thing being tested _should_ break the tests, or at least the one test that tests that thing.

Interface and implementation are terms in common usage in the industry on this subject and I don't think we should avoid using them.

tbh I'm speculating on what people with less experience in testing might need the wording of this to say. I totally agree with the sentiment of the commandment. It would be useful to get wider feedback from other devs on their reading of it.